cmake/modules/MOLD.cmake

```diff
@@ -15,6 +15,28 @@ if(ENABLE_MOLD)
     set(COMPILER_ARGS "-fuse-ld=mold")
   endif()
 
+  set(DEFAULT_ENABLE_DEBUGFISSION ON)
+  include(CMakeDependentOption)
+  cmake_dependent_option(ENABLE_DEBUGFISSION "Enable Debug Fission support" ON
+                         "DEFAULT_ENABLE_DEBUGFISSION" OFF)
+
```

Comment:
This is kind of pointless if you are setting DEFAULT_ENABLE_DEBUGFISSION in the first line.

Reply:
iirc, my original intent was to split ENABLE_DEBUGFISSION out as a global option. Any linker/compiler combo can then use that if supported.

Reply:
this is addressed by the first fixup which default enables DEBUGFISSION for Debug and RelWithDebInfo like it is done for ld.gold. guess wsnipex you don't like duplicating this, i can try to sort it out as a global option at a later point. actually it is a bit over my head with cmake :)